### PR TITLE
Remove unwanted kernel telemetry events

### PR DESCRIPTION
**Pull request by DonJayamanne**

### CODE REVIEW

File: src/platform/telemetry/index.ts [hunk 1]
```diff
@@ -385,12 +385,6 @@ export class SharedPropertyMapping {
      */
     ['isamlcompute']: 'true' | 'false';
 
-    /**
-     * For every telemetry event from the extension we want to make sure we can associate it with install
-     * source. We took this approach to work around very limiting query performance issues.
-     */
-    ['installSource']: undefined | 'marketPlace' | 'pythonCodingPack';
```
Comment (DonJayamanne) [inline src/platform/telemetry/index.ts:392]: Not required

File: src/platform/common/constants.ts [hunk 2]
```diff
@@ -502,11 +500,7 @@ export enum Telemetry {
     IPyWidgetOverhead = 'DS_INTERNAL.IPYWIDGET_OVERHEAD',
     IPyWidgetRenderFailure = 'DS_INTERNAL.IPYWIDGET_RENDER_FAILURE',
     IPyWidgetUnhandledMessage = 'DS_INTERNAL.IPYWIDGET_UNHANDLED_MESSAGE',
-    JupyterCreatingNotebook = 'DS_INTERNAL.JUPYTER_CREATING_NOTEBOOK',
-    RawKernelSessionConnect = 'DS_INTERNAL.RAWKERNEL_SESSION_CONNECT',
-    RawKernelStartRawSession = 'DS_INTERNAL.RAWKERNEL_START_RAW_SESSION',
```
Comment (DonJayamanne) [inline src/platform/common/constants.ts:507]: These are either duplicates or not required

File: src/kernels/telemetry/sendKernelTelemetryEvent.ts [hunk 3]
```diff
@@ -33,36 +30,6 @@ export function sendKernelTelemetryEvent<P extends IEventNamePropertyMapping, E
...
         })
         .ignoreErrors();
 }
-
-/**
- * Some information such as interrupt counters & restart counters need to be reset
- * after we have successfully interrupted or restarted a kernel.
- */
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-function resetData(resource: Resource, eventName: string, properties: any) {
```
Comment (DonJayamanne) [inline src/kernels/telemetry/sendKernelTelemetryEvent.ts:48]: No longer required

File: src/kernels/telemetry/helper.ts [hunk 4]
```diff
@@ -172,25 +150,3 @@ export async function initializeInteractiveOrNotebookTelemetryBasedOnUserAction(
 ) {
     await trackKernelResourceInformation(resourceUri, { kernelConnection, userExecutedCell: true });
 }
-
-export function clearInterruptCounter(resource: Resource) {
```
Comment (DonJayamanne) [inline src/kernels/telemetry/helper.ts:176]: No longer required

**Review by joaomoreno — APPROVED** [review]

**Pull request merged**